cosmos/models/Stage.py

```diff
@@ -14,7 +14,9 @@
 
 @signal_stage_status_change.connect
 def stage_status_changed(stage):
-    stage.log.info('%s %s (%s/%s Tasks were successful)' % (stage, stage.status, sum(t.successful for t in stage.tasks), len(stage.tasks)))
```

Comment:
@egafni For some reason the changes I made elsewhere made this line break. The patch below makes the tests pass, but it _clearly_ isn't the right answer. For some reason `t.successful` is now `None` for at least one test task when Cosmos starts up. See https://travis-ci.org/LPM-HMS/COSMOS2/builds/217048614 for a trace. I'm stumped; the changes I made don't touch `t.successful` to my knowledge.